### PR TITLE
[UI][Adrian] 404-coming-soon

### PR DESCRIPTION
**Pull request by Wetooa**
pa check pls. need feedback uwu

### CODE REVIEW

File: pages/404.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import Header from '@/components/Header';
+import Section from '@/components/Section';
+import Link from 'next/link';
+import Image from 'next/image';
+import TakeMeHomeButton from '@/components/TakeMeHomeButton';
+
+export default function pageNotFound() {
```
Comment (maxellmilay) [inline pages/404.tsx]: I think the naming for components would be in pascal case, so this should be `PageNotFound()`

File: pages/coming-soon.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import Header from '@/components/Header';
+import Section from '@/components/Section';
+import Link from 'next/link';
+import Image from 'next/image';
+import TakeMeHomeButton from '@/components/TakeMeHomeButton';
+
+export default function comingSoon() {
```
Comment (maxellmilay) [inline pages/coming-soon.tsx]: same naming convention here

File: pages/404.tsx (hunk 1 above)
Comment (Wetooa) [reply pages/404.tsx]: ahh ic ic

File: pages/coming-soon.tsx (hunk 2 above)
Comment (Wetooa) [reply pages/coming-soon.tsx]: thx will tend to changes

File: components/TakeMeHomeButton.tsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import Link from 'next/link';
+
+export default function TakeMeHomeButton() {
+	return (
+		<>
```
Comment (ezerssss) [inline components/TakeMeHomeButton.tsx]: pwede rani tangalon ang `<>`

File: pages/index.tsx [hunk 4]
```diff
@@ -1,48 +1,48 @@
...
-    <>
-      <Header />
-      <Section className="bg-[url('/images/latest-sermon.png')] no-repeat bg-cover bg-top pt-96 sm:pt-80">
-        <h1 className="text-3xl font-thin">LATEST SERMON</h1>
-        <p className="font-bold md:text-6xl sm:text-4xl text-xl mb-8 mt-5">
-          The Premise of the Promise <br /> of a Happy New Year
-        </p>
-        <button className="bg-tmi-yellow py-1.5 px-7 font-bold rounded-3xl mr-3 mb-3">
-          Watch Now
-        </button>
```
Comment (ezerssss) [inline pages/index.tsx:16]: oops, make sure nga naka install ka sa prettier nga extension

**Review by ezerssss — CHANGES_REQUESTED** [review]

File: components/TakeMeHomeButton.tsx (hunk 3 above)
Comment (Wetooa) [reply components/TakeMeHomeButton.tsx]: ok will do

File: pages/index.tsx (hunk 4 above)
Comment (Wetooa) [reply pages/index.tsx:16]: already fixed local issue

**Review by ezerssss — APPROVED** [review]

**Pull request merged**